reference_controllers/rabbitmq_controller_unverified/src/rabbitmqcluster_status.rs

```diff
@@ -0,0 +1,12 @@
+use k8s_openapi::api::core::v1 as corev1;
+use kube::CustomResource;
+use schemars::JsonSchema;
+use serde::{Deserialize, Serialize};
+use std::vec;
+
+pub struct RabbitmqClusterStatus {}
```

Comment:
Seems that `RabbitmqClusterStatus` is not used anywhere?

Reply:
Yes. I left them here for later update features. Or do I need to delete them?